CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # [Versions](https://mui.com/versions/)
 
+## 7.0.0
+
+<!-- generated comparing v7.0.0-rc.0..master -->
+
+_Mar 26, 2025_
+
+🎉 We're happy to announce the stable release of v7.
+
+- Read the blog post [Material-UI v7 stable release](https://mui.com/blog/material-ui-v7-is-here/).
+- Check out the [v7 upgrade guide](https://mui.com/material-ui/migration/upgrade-to-v7/).
+
+A big thanks to the 10 contributors who made this release possible.
```

Comment:
```suggestion
```

Maybe we can drop this section from the announcing. It's been more than 10 people who made the stable possible :)